lib/tasks/elasticsearch_reindex.rake

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Executes an `update_index` call on all ElasticSearch indexed models.
+task elasticsearch_reindex: :environment do
+  task_stats = {}
+  Chewy.strategy(:active_job) do
+    ESIndices.models.each do |model|
+      model_name = model.to_s.underscore
+      task_stats[model.to_s] = 0
+      model.find_in_batches(batch_size: 100) do |model_instances|
+        model_instances.each do |model_instance|
+          model.update_index("#{model_name}##{model_name}") { model_instance }
```

Comment:
Will this run inline or in a background job?

If it runs inline and there's an exception this will cause a break. If it's in the background, you'll just end up with some failed jobs when exceptions are raised.

Reply:
The indexing will happen in a background job. 
If indexing fails, it will be retried with exponential backoff (default in sidekick).